lib/models/task.dart

```diff
@@ -0,0 +1,18 @@
+class Task {
+  final int id;
+  final String description;
+  final String category;
+  final int clock;
```

Comment:
We should probably change clock to something more representative of time @haltaf19 @Fandrew44  LOL